bundles/org.openhab.transform.basicprofiles/README.md

```diff
@@ -180,31 +180,81 @@ Switch motionSensorFirstFloor {
+When `ITEM_NAME` is omitted, e.g. `> 10, < 100`, the comparisons are applied against the input data from the binding.
+In this case, the value can also be replaced with an item name, which will result in comparing the input state against the state of that item, e.g. `> LowerLimitItem, < UpperLimitItem`.
+This can be used to filter out unwanted data, e.g. to ensure that incoming data are within a reasonable range.
 
-| Configuration Parameter | Type | Description                                                                                                                                                                                       |
-|-------------------------|------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `conditions`            | text | Comma separated list of expressions on the format `ITEM_NAME OPERATOR ITEM_STATE`, ie `MyItem EQ OFF`. Use quotes around `ITEM_STATE` to treat value as string ie `'OFF'` and not `OnOffType.OFF` |
-| `mismatchState`         | text | Optional state to pass instead if conditions are NOT met. Use single quotes to treat as `StringType`. Defaults to `UNDEF`                                                                         |
-| `separator`             | text | Optional separator string to separate expressions when using multiple. Defaults to `,`                                                                                                            |
+Some tips:
 
-Possible values for token `OPERATOR` in `conditions`:
+- When dealing with QuantityType data, the unit must be included in the comparison value, e.g.: `PowerItem > 1 kW`.
+- Use single quotes around the `VALUE` to perform a string comparison, e.g. `'UNDEF'` is not equal to `UNDEF` (of type `UnDefType`).
+  This will distinguish between a string literal and an item name or a constant such as `UNDEF`, `ON`/`OFF`, `OPEN`, etc.
+- `VALUE` cannot be on the left hand side of the operator.
 
-- `EQ` - Equals
-- `NEQ` - Not equals
+##### State Filter Operators
 
+| Name  | Symbol |                           |
+| :---: | :----: | ------------------------- |
+| `EQ`  |  `==`  | Equals                    |
+| `NEQ` |  `!=`  | Not equals                |
+| `GT`  |  `>`   | Greater than              |
+| `GTE` |  `>=`  | Greater than or equals to |
+| `LT`  |  `<`   | Less than                 |
+| `LTE` |  `<=`  | Less than or equals to    |
 
-### Full Example
+Notes:
+
+- The operator names must be surrounded by spaces, i.e.: `Item EQ 10`
+- The operator symbols do not need to be surrounded by spaces, e.g.: `Item==10` and `Item == 10` are both fine.
+- Only symbolic operators can be used when comparing against the incoming state, e.g. `> 10`. Using operator names isn't supported, i.e. this is not supported: ~~`GT 10`~~.
```

Comment:
Why is that?

Reply:
The regex is more complicated to support it, but I've redone it now.